### ISSUE TITLE
Request text too long, text length cannot exceed 1000

### ISSUE THREAD

**Issue by WAILMAGHRANE** [HeyGen-Official/StreamingAvatarSDK#22]
Hi, does anyone have any ideas or solutions to increase the response token length?

**Comment by kauly** [comment]
I dont know how to increase but I'm using the `system` prop of the `ai` package. The result is pretty good.

```js
  const result = await streamText({
    model: openaiModel("gpt-4"),
    system:
      `You are a virtual assistant.` +
      `Your answers can not have more than 100 words.`,
    messages,
  });
```

**Comment by WAILMAGHRANE** [comment]
Yes, it's good, but I have a report that exceeds 1000 tokens, and I need to increase the token limit to accommodate the entire report.

> I dont know how to increase but I'm using the `system` prop of the `ai` package to limit the response. The result is pretty good.
> 
> ```js
>   const result = await streamText({
>     model: openaiModel("gpt-4"),
>     system:
>       `You are a virtual assistant.` +
>       `Your answers can not have more than 100 words.`,
>     messages,
>   });
> ```

Yes, it's good, but I have a report that exceeds 1000 tokens, and I need to increase the token limit to accommodate the entire report.
